contract/views/contract_template_line.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think you removed this file unintentionally.

Reply:
No, in my understanding, it was useless.

Reply:
Without this inheritance, contract template lines have no default form view.
If you take a look at runbot, when I add a line to a contract template, I get this view with all model fields.
Impossibility to save as contract template is required.
![1](https://user-images.githubusercontent.com/12665409/58544631-ee885100-8201-11e9-87de-bf96b6948b2e.png)

Reply:
Ok! Thanks.